index.js

```diff
@@ -68,10 +68,11 @@ module.exports = function(version, _options) {
 
             return config.join('');
         },
-        compile: function(language, step) {
+        compile: function(language, step, legIndex, legCount) {
```

Comment:
Can we make this optional? For most cases (read: non-multi waypoint cases) we will not need these properties. I'd be good to put them into an options object.

```js
compile: function(language, step, options) {
```